pkg/api/controller.go

```diff
@@ -3493,6 +3493,39 @@ func (c *Controller) GetTag(w http.ResponseWriter, r *http.Request, repository s
 	writeResponse(w, r, http.StatusOK, response)
 }
 
+func makeLoginConfig(c *config.Config) *LoginConfig {
+	// TODO(ariels): Configure by c.Auth.OIDC.Enabled if set, otherwise
+	// from C.Auth.UIConfig
+	var (
+		cookies = c.Auth.UIConfig.LoginCookies
+		// []string{
+		// 	"internal_auth_session",
+		// }
+		loginFailedMessage = c.Auth.UIConfig.LoginFailedMessage // "The credentials don't match."
+		fallbackLoginURL   = c.Auth.UIConfig.FallbackLoginURL   // nil
+		fallbackLoginLabel = c.Auth.UIConfig.FallbackLoginLabel // nil
```

Comment:
Are the comments here the default values? Because I don't see them set anywhere.